pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -1,9 +1,28 @@
-{ lib, stdenv, fetchurl, fetchzip, unzip }:
+{ lib, stdenv, fetchurl, fetchzip, zip, unzip, xmlstarlet, eclipses }:
 
 rec {
 
-  # A primitive builder of Eclipse plugins. This function is intended
-  # to be used when building more advanced builders.
+  sanitizeUpdateSite = outDir: ''
```

Comment:
This feels a bit ad hoc.

Both these jars just contain a single xml file, and the xml files contain an attribute with a timestamp, so we:
- extract
- zero the xml timestamp with xmlstarlet
- clear file timestamps
- rezip

Is there a cleaner way to do any of this? Existing utilities in nixpkgs to help?